src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9922,11 +9922,11 @@ static ColliderQuadInit D_808546A0 = {
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
 
-void func_8084663C(Actor* thisx, PlayState* play) {
+void Player_DoNothing(Actor* thisx, PlayState* play) {
 }
 
-void func_80846648(PlayState* play, Player* this) {
-    this->actor.update = func_8084663C;
+void Player_SetupDoNothing(PlayState* play, Player* this) {
+    this->actor.update = Player_DoNothing;
     this->actor.draw = NULL;
 }
```

Comment:
These functions in player specifically are "initMode" functions, that control how player "loads into" the world. I think this should be documented with the rest of the initModes. I would personally revert all these changes in player.